google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -186,14 +194,22 @@ private void resetRuntimeDisplay() {
     }
   }
 
+  private void updateStagedArtifactNameEmptyText() {
+    Path artifactPath = Paths.get(Strings.nullToEmpty(deploymentSource.getFilePath()));
+    AppEngineFlexibleDeploymentArtifactType deploymentArtifactType =
+        AppEngineFlexibleDeploymentArtifactType.typeForPath(artifactPath);
+    stagedArtifactNameTextField
+        .getEmptyText()
+        .setText(deploymentArtifactType.getDefaultArtifactName());
+  }
+
   private void reloadAppYamls(Project project) {
-    appYamlCombobox.setModel(new DefaultComboBoxModel<>(
```

Comment:
This is also a note to self, but in the future for PRs that have a lot of reformatting it could be helpful to first send a PR with just the file reformatting, followed by the PR with the actual logic changes. Makes it easier to tell what has or hasn't non-trivially changed.